BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -634,6 +634,9 @@ public void InitializationIsSkippedForSampledItem(string appDomainName = "Incorr
         [Event(67, Message = "Backend has responded with {0} status code in {1}ms.", Level = EventLevel.Informational)]
         public void IngestionResponseTime(int responseCode, float responseDurationInMs, string appDomainName = "Incorrect") => this.WriteEvent(67, responseCode, responseDurationInMs, this.nameProvider.Name);
 
+        [Event(68, Message = "Activity isn't available.", Level = EventLevel.Error)]
+        public void ActivityNotAvailable(string exception, string appDomainName = "Incorrect") => this.WriteEvent(68, exception, this.nameProvider.Name);
```

Comment:
This won't work. check out some of the other examples of passing an exception to the EventSource.

Reply:
@TimothyMothra , we have some patters, so i got confused:
- some places use exception.ToString()
- some places use exception.Message
- some places use exception.ToInvariantString()

After that, we have places that do (in CoreEventSource.cs):
- WriteEvent(number, error, this.nameProvider.name)
- WriteEvent(number, error ?? string.Empty, this.nameProvider.name)

Reply:
where the exception is caught, you should call ToInvariantString().
This will pass a string into the EventSource message which needs to be included in the Message attribute.

Reply:
ok! thanks for the update!